src/Features/Core/Portable/UseSystemHashCode/UseSystemHashCodeCodeFixProvider.cs

```diff
@@ -43,7 +43,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             SyntaxEditor editor, CancellationToken cancellationToken)
         {
             var generator = SyntaxGenerator.GetGenerator(document);
-            var declarationService = document.GetLanguageService<ISymbolDeclarationService>();
+            var declarationService = document.GetLanguageService<ISymbolDeclarationService>()!;
```

Comment:
You can do document.Project.LanguageServices.GetRequiredService() which returns non-nullable (and throws if ever this is somehow broken.)

(Although I'm not understanding why we're seeing this in this PR -- is GitHub acting funny?)

Reply:
Same comment as above.

Reply:
looks like a bad merge?